File: followingApp/followingApp/app/src/main/java/com/example/idolmastercalendar/ViewPagerAdapter.java
package com.example.idolmastercalendar;

import androidx.annotation.NonNull;
import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentActivity;
import androidx.viewpager2.adapter.FragmentStateAdapter;

import java.util.ArrayList;

public class ViewPagerAdapter extends FragmentStateAdapter {

    private ArrayList<Fragment> fragmentArrayList = new ArrayList<>();

    public ViewPagerAdapter(FragmentActivity fragmentActivity) {
        super(fragmentActivity);

        fragmentArrayList.add(ConceptFragment.newInstance());
        fragmentArrayList.add(IdolFragment.newInstance());
    }

    @NonNull
    @Override
    public Fragment createFragment(int position) {
        return fragmentArrayList.get(position);
    }

    @Override
    public int getItemCount() {
        return fragmentArrayList.size();
    }
}

File: WimiProject/WimiProject/app/src/main/java/com/wimi/idolmaster/ui/choose/ViewPagerAdapter.kt
package com.wimi.idolmaster.ui.choose

import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentActivity
import androidx.viewpager2.adapter.FragmentStateAdapter
import com.wimi.idolmaster.ui.choose.concept.ConceptFragment
import com.wimi.idolmaster.ui.choose.idol.IdolFragment
import java.util.*

class ViewPagerAdapter(fragmentActivity: FragmentActivity) : FragmentStateAdapter(fragmentActivity) {
    private val fragmentArrayList = ArrayList<Fragment>()

    init {
        fragmentArrayList.add(ConceptFragment())
        fragmentArrayList.add(IdolFragment())
    }

    override fun createFragment(position: Int): Fragment {
        return fragmentArrayList[position]
    }

    override fun getItemCount(): Int {
        return fragmentArrayList.size
    }
}